graphql/schemabuilder/input.go

```diff
@@ -98,11 +98,6 @@ func (sb *schemaBuilder) makeStructParser(typ reflect.Type) (*argParser, graphql
 					return fmt.Errorf("%s: %s", name, err)
 				}
 			}
-			for name := range asMap {
```

Comment:
Do you also add a comment to say why it is safe to remove this check?

Reply:
added to the commit message!